src/cunumeric/unary/unary_op_util.h

```diff
@@ -1181,4 +1183,37 @@ struct UnaryOp<UnaryOpCode::TRUNC, legate::LegateTypeCode::HALF_LT> {
   }
 };
 
+template <UnaryOpCode OP_CODE, legate::LegateTypeCode CODE>
+struct MultiOutUnaryOp {
+  static constexpr bool valid = false;
+};
+
+template <legate::LegateTypeCode CODE>
+struct MultiOutUnaryOp<UnaryOpCode::FREXP, CODE> {
+  static constexpr bool valid = legate::is_floating_point<CODE>::value;
+  using RHS1                  = legate::legate_type_of<CODE>;
+  using RHS2                  = int32_t;
+  using LHS                   = RHS1;
```

Comment:
Shouldn't it be LHS1, LHS2 and RHS?

Reply:
I initially wanted to do that, but the c++ functions we're calling actually have two RHS', so I think RHS1 and RHS2 would be more natural.